generators/kubernetes/templates/registry/_jhipster-registry.yml

```diff
@@ -84,17 +84,13 @@ spec:
         # Registry configuration
         - name: SPRING_PROFILES_ACTIVE
           value: prod,native
-        - name: SECURITY_USER_PASSWORD
+        - name: JHIPSTER_REGISTRY_PASSWORD
           valueFrom:
             secretKeyRef:
               name: registry-secret
               key: registry-admin-password
         - name: JHIPSTER_SECURITY_AUTHENTICATION_JWT_SECRET
           value: my-secret-token-to-change-in-production
-        - name: EUREKA_CLIENT_FETCH_REGISTRY
-          value: 'true'
-        - name: EUREKA_CLIENT_REGISTER_WITH_EUREKA
-          value: 'true'
```

Comment:
I am not sure here. The [official documentation](http://cloud.spring.io/spring-cloud-netflix/spring-cloud-netflix.html) doesn't say this is needed. 
Furthermore, you definitely see it is not running in peer-aware when reloading Eureka gives distinct view to 2 registries, rather than a consistent list of instances. In contraposition, if you reload instances and always get the same list, it means the Eureka instances are replicated properly.

So the only thing "register with Eureka" is doing, is the self-registration, which is IMHO not correct.